account_mass_reconcile_ref_deep_search/tests/test_account_reconcile_ref_deep_search.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.tests import SavepointCase
+
+
+class TestAccountReconcileRefDeepSearch(SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestAccountReconcileRefDeepSearch, cls).setUpClass()
```

Comment:
not mandatory but having `cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))` would be nice